credit_card_integration_test.go

```diff
@@ -5,6 +5,8 @@ import (
 )
 
 func TestCreditCard(t *testing.T) {
+	// TODO: Fix integration - Update ends with 404 Not Found ✓
+	t.Skip()
```

Comment:
Did the original version code act the same?
I am talking about all skipped tests.

Reply:
The original version didn't actually compile without changes (even when I tried with go1.6.4).
Not sure yet why, but bot the following error:
```
Cloning into 'braintree-go/src/gopkg.in/yaml.v3'...
fatal: unable to access 'https://gopkg.in/yaml.v3/': server certificate verification failed. CAfile: /etc/ssl/certs/ca-certificates.crt CRLfile: none
```

When tried to run with newer go version (go1.21.1) using `GO111MODULE=off` (no module compatibility), I could run tests and these failed with the same results as in the submitted version.

Reply:
I think Petro meant why do we need to skip those tests - and whether those new TODOs could be resolved? Or those are around things we are not using?

Also worth checking - whether those tests failed previously (on master etc.)

Reply:
Well, I couldn't run tests on master, because without any changes, I couldn't even compile the project on master.
Maybe someone might try to compile and run tests with entire repo untouched, but I didn't manage to.

After having added go module, project compiled and these tests failed, as described here - [Slack thread](https://processout.slack.com/archives/C03EQGZR56K/p1695286148401739?thread_ts=1695211263.901599&cid=C03EQGZR56K).

Adding go module was just a set of commands:
```sh
go mod init github.com/processout/braintree-go
go mod vendor
```

And then, running `go test ./...` failed with the failures I mentioned before.